voting/src/voting.rs

```diff
@@ -285,6 +286,30 @@ impl VoteManager for Context {
 
         Ok(vote_paper.number_of_shares)
     }
+
+    fn get_number_of_agenda(&self, vote_id: &VoteId) -> Result<u32, Error> {
```

Comment:
I feel like `number of agenda` means the maximum number of agenda.
`agenda number` feels like the index number of the agenda.

How do you feel when you read the name?

If the name confuses us, we may use totally different names for them. Like `agenda_id`?